### PR TITLE
NXBT-3515: Deploy a NFS server for RWX volumes

### PR DESCRIPTION
**Pull request by kevinleturc**

### CODE REVIEW

File: charts/nfs-server-provisioner/values.yaml.gotmpl [hunk 1]
```diff
@@ -0,0 +1,14 @@
+persistence:
+  enabled: true
```
Comment (ataillefer) [inline charts/nfs-server-provisioner/values.yaml.gotmpl:2]: needed for preview?

File: charts/nfs-server-provisioner/values.yaml.gotmpl [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  storageClass: standard
+  size: 500Gi
+storageClass:
+  name: {{ .Values.namespace }}-nfs
+  provisionerName: cluster.local/{{ .Values.namespace }}-nfs
```
Comment (ataillefer) [inline charts/nfs-server-provisioner/values.yaml.gotmpl:7]: is this a convention?

File: charts/nfs-server-provisioner/values.yaml.gotmpl [hunk 3]
```diff
@@ -0,0 +1,14 @@
+persistence:
+  enabled: true
+  storageClass: standard
```
Comment (ataillefer) [inline charts/nfs-server-provisioner/values.yaml.gotmpl:3]: not the default?

File: charts/nfs-server-provisioner/values.yaml.gotmpl (hunk 2 above)
Comment (kevinleturc) [reply charts/nfs-server-provisioner/values.yaml.gotmpl:7]: For the _cluster.local_ part it seems to be yes, it is also the prefix in the default configuration.

I put the the part after the `/` in order to know how the provisionner will be called.

File: charts/nfs-server-provisioner/values.yaml.gotmpl (hunk 3 above)
Comment (kevinleturc) [reply charts/nfs-server-provisioner/values.yaml.gotmpl:3]: Yes it is, I'll remove it.

**Review by ataillefer — APPROVED** [review]
:rocket:

**Pull request merged**